packages/core/src/components/Menu/index.tsx

```diff
@@ -74,11 +74,13 @@ export default withStyles(({ color, ui, pattern }) => ({
 
       // These are jank. Better way?
       '> li:first-child > *': {
+        borderTop: `${unit}px solid ${color.accent.bg}`,
```

Comment:
8px border? Is that accurate?

Reply:
Yeah, it's for top/bottom spacing around menu items:
<img width="238" alt="Screen Shot 2019-07-23 at 10 51 05 AM" src="https://user-images.githubusercontent.com/8676510/61734979-d59aa880-ad37-11e9-9b50-44b2d3480548.png">

You can see in the screenshot from the initial `Description` that this results in some weird focus, but it also means all parts of the Menu are clickable. It's kind of a weird thing, and I'd be ok using non-interactive padding, but that's basically the tradeoff.

Reply:
On second thought, maybe this should just be top/bottom padding on `menu`?

Reply:
Awesome, got it
<img width="240" alt="Screen Shot 2019-07-23 at 4 35 15 PM" src="https://user-images.githubusercontent.com/8676510/61754393-de0ad780-ad67-11e9-87f1-fc421afdc22c.png">